pyseir/deployment/webui_data_adaptor_v1.py

```diff
@@ -393,9 +417,10 @@ def generate_state(self, whitelisted_county_fips: list, states_only=False):
 
 
 if __name__ == "__main__":
+    dataset_cache.set_pickle_cache_dir()
     # Need to have a whitelist pre-generated
     # Need to have state output already built
     mapper = WebUIDataAdaptorV1(
-        state="California", output_interval_days=4, run_mode="can-inference-derived"
+        state="California", output_interval_days=1, run_mode="can-inference-derived"
```

Comment:
Why did you change this?

Reply:
It's always overwritten when we instantiate this class, and we use 1 in the api.

Reply:
It's also easier to reason about the spacing of the timeseries lengths with a day=1 spacing. If someone wants to revert back to 4 day spacing they can make it with an explicit call.